examples/models/llama2/CMakeLists.txt

```diff
@@ -0,0 +1,226 @@
+
+# Find prebuilt libraries. executorch package should contain
+# portable_ops_lib, etdump, bundled_program.
+find_package(executorch CONFIG REQUIRED)
+target_include_directories(executorch INTERFACE ${_common_include_directories})
+target_compile_options(executorch INTERFACE -DET_EVENT_TRACER_ENABLED)
+
+find_package(
+  gflags REQUIRED PATHS ${CMAKE_CURRENT_BINARY_DIR}/../../../third-party
+)
+
+# portable_ops_lib
+include(${EXECUTORCH_ROOT}/build/Utils.cmake)
+include(${EXECUTORCH_ROOT}/build/Codegen.cmake)
+gen_selected_ops("" "" "ON")
+generate_bindings_for_kernels(
+  ${EXECUTORCH_ROOT}/kernels/portable/functions.yaml ""
+)
+gen_operators_lib("portable_ops_lib" portable_kernels executorch)
+
+set(llama_runner_libs "-framework Foundation")
+
+#
+# The `_<target>_srcs` lists are defined by including ${EXECUTORCH_SRCS_FILE}.
+#
+set(
+  EXECUTORCH_SRCS_FILE
+  "${CMAKE_CURRENT_BINARY_DIR}/../../../executorch_srcs.cmake"
+)
+
+extract_sources(${EXECUTORCH_SRCS_FILE})
+
+
+add_library(extension_module "${CMAKE_CURRENT_SOURCE_DIR}/../../../extension/module/module.cpp")
+target_include_directories(extension_module PUBLIC ${EXECUTORCH_ROOT}/..)
+
+#add_library(extension_data_loader "${CMAKE_CURRENT_SOURCE_DIR}/../../../extension/data_loader/mmap_data_loader.cpp")
+#target_link_libraries(extension_data_loader executorch)
+
+
+# Install libraries
+install(
+  TARGETS extension_module
+  DESTINATION ${CMAKE_BINARY_DIR}/lib
+  INCLUDES
+  DESTINATION ${_common_include_directories})
+
+
+set(llama_schema_headers ${CMAKE_BINARY_DIR}/../../../schema/include/)
+include(${EXECUTORCH_SRCS_FILE})
+target_include_directories(
+  bundled_program
+  INTERFACE
+  ${CMAKE_CURRENT_BINARY_DIR}/../../../sdk/include
+  ${CMAKE_CURRENT_BINARY_DIR}/../../../sdk/bundled_program
+  ${EXECUTORCH_ROOT}/third-party/flatbuffers/include
+  ${EXECUTORCH_ROOT}/third-party/flatcc/include
+  ${llama_schema_headers}
+)
+
+function(define_common_targets)
+    set(EXPORTED_HEADERS_RUNNER
+        "${CMAKE_CURRENT_SOURCE_DIR}/runner/runner.h"
+        "${CMAKE_CURRENT_SOURCE_DIR}/runner/util.h"
+    )
+
+    # Define a CMake target using runtime.cxx_library
+    add_library(runner
+        "${CMAKE_CURRENT_SOURCE_DIR}/runner/runner.cpp"
+        "${CMAKE_CURRENT_SOURCE_DIR}/sampler/sampler.cpp"
+        "${CMAKE_CURRENT_SOURCE_DIR}/tokenizer/tokenizer.cpp"
+    )
+
+    # Specify exported headers
+    target_include_directories(runner PUBLIC
+        ${CMAKE_CURRENT_SOURCE_DIR}/../../../..
+        ${CMAKE_CURRENT_SOURCE_DIR}/runner
+        ${CMAKE_CURRENT_SOURCE_DIR}/sampler
+        ${CMAKE_CURRENT_SOURCE_DIR}/tokenizer
+        ${CMAKE_CURRENT_SOURCE_DIR}/../../../third-party/pytorch/torch/csrc/api/include
+        ${CMAKE_CURRENT_SOURCE_DIR}/../../../third-party/pytorch
+        ${CMAKE_CURRENT_SOURCE_DIR}/../../../libtorch/include/
+        ${CMAKE_CURRENT_SOURCE_DIR}/../../../libtorch/include/torch/csrc/api/include/
+    )
```

Comment:
Same comment as above. As for libtorch include, I don't think we need those?

Reply:
Without these, I got error including:
<img width="1399" alt="image" src="https://github.com/pytorch/executorch/assets/535090/a9e1243e-f067-4836-afa8-2ef2f5f92134">